frang/test_http_resp_code_block.py

```diff
@@ -0,0 +1,62 @@
+"""Functional tests for http_resp_code_block."""
+
+import os
+
+from framework import tester, deproxy_server
+from helpers import tf_cfg, deproxy, tempesta, control
+from framework.templates import fill_template
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2019 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+class HttpRespCodeBlock(tester.TempestaTest):
+    backends = [
+        {
+            'id' : 'deproxy',
+            'type' : 'deproxy',
+            'port' : '8000',
+            'response' : 'static',
+            'response_content' : """HTTP/1.1 404 OK
```

Comment:
Not very critical, but ,please, always generate messages that looks usual. E.g. `404` code is not  `OK`, it's `Not found`.